docs/marionette.behavior.md

```diff
@@ -316,10 +321,23 @@ including:
 * [`modelEvents`](./marionette.view.md#model-events)
 * [`collectionEvents`](./marionette.view.md#collection-events)
 
+### Events Order
+
+If both view and behavior are listening for the same event, this will be executed
+first in the view than in the behavior. 
+
+Unlike events, the initialize method will 
+be called first in the behavior. 
+  
+> While initialize is called in behavior before than in view, the view property is already set, and can be accessed, inside behavior initialize 
+  
+[Live example](https://jsfiddle.net/9wtqgwgu/)  
```

Comment:
While initialize is called in behavior before than in view, the view property is already set, and can be accessed, inside behavior initialize 

The above sentence, is not very good. I tried various versions. If someone has suggestions, please do.